retrace/glstate_images.cpp

```diff
@@ -1727,10 +1744,20 @@ dumpFramebufferAttachment(StateWriter &writer, Context &context, GLenum target,
         glGetFramebufferAttachmentParameteriv(target, attachment,
                                               GL_FRAMEBUFFER_ATTACHMENT_LAYERED,
                                               &layered);
-        if (layered &&
-            isGeometryShaderBound(context)) {
+
+        // OVR_multiview
+        GLint first_view = 0;
```

Comment:
Sorry.  Somehow my comment here got lost.

Have you tested this on a system that doesn't support `GL_OVR_multiview`?  I suspect this will trigger GL errors and cause troubles.

This code should be predicated by a `GL_OVR_multiview` extension check,

Reply:
Yes, good spot. I had assumed the errors would be harmlessly ignored, but if I use glFramebufferTexture() and run without OVR_multiview, and then replay up to a draw command, GL_INVALID_ENUMs show up multiple times in the error log.
I'm guessing from a quick dig that OVR_multiview should be a new bit in struct Features, that can then be accessed from the context object.